### PR TITLE
{chem}[foss/2020a] MRChem v1.0.0 w/ Python-3.8.2

### PR DESCRIPTION
**Pull request by stigrj**

### CODE REVIEW

**Comment by stigrj** [comment]
I don't understand the error

File: easybuild/easyconfigs/m/MRChem/MRChem-1.0.0-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('Eigen', '3.3.7'),
+    ('MRCPP', '1.3.6'),
+    ('Python', '3.8.2'),
+    ('XCFun', '2.1.0')
+]
+
+dependencies = [
+    ('MRCPP', '1.3.6'),
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MRChem/MRChem-1.0.0-foss-2020a-Python-3.8.2.eb:28]: @stigrj there's no need to include `MRCPP` and `Python` as `builddependencies` if you have them as `dependencies`, so please remove them from the former.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/MRChem/MRChem-1.0.0-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (stigrj) [reply easybuild/easyconfigs/m/MRChem/MRChem-1.0.0-foss-2020a-Python-3.8.2.eb:25]: Thanks @verdurin!

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
@stigrj would be good if you could submit a test build for this.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.6
See https://gist.github.com/0d108d0f0da2836d800e5e78b94526fe for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3575.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/8acd364431205d01dffd9819fe2c01bb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2407.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/138480234e3f8eb83e6609a7086990e0 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @stigrj!

**Pull request merged**